demisto_sdk/commands/prepare_content/integration_script_unifier.py

```diff
@@ -551,17 +552,20 @@ def insert_pack_version(
             existing_pack_info_debug_log_re = (
                 r"demisto\.debug\('pack name = .*?, pack version = .*?'\)"
             )
-            pack_name_escaped = pack_name.replace("'", "\\'")
-            pack_info_debug_statement = f"demisto.debug('pack name = {pack_name_escaped}, pack version = {pack_version}')"
+            global_name = f"{pack_id.upper()}_PACK_VERSION"
+            pack_version_statement = (
+                f"{global_name} = '{pack_version}'"
+                f"\ndemisto.debug(f'pack id = {pack_id}, pack version = {{{global_name}}}')"
```

Comment:
Shouldn't it be only 2?
```suggestion
                f"\ndemisto.debug(f'pack id = {pack_id}, pack version = {{global_name}}')"
```

Reply:
no, with two, will it print "pack id = the_pack_id, pack version = {global_name}"

Reply:
r u sure?

Reply:
<img width="655" alt="Screenshot 2025-03-10 at 13 07 15" src="https://github.com/user-attachments/assets/cf12c275-8263-4b8c-8099-ebbfa8909436" />